build-tools/create-packs/Microsoft.Android.Ref.proj

```diff
@@ -33,6 +33,7 @@ by projects that use the Microsoft.Android framework in .NET 6+.
     <ItemGroup>
       <_AndroidRefPackAssemblies Include="$(JavaInteropSourceDirectory)\bin\$(Configuration)-net7.0\ref\Java.Interop.dll" />
       <_AndroidRefPackAssemblies Include="$(_MonoAndroidNETDefaultOutDir)ref\Mono.Android.dll" />
+      <_AndroidRefPackAssemblies Include="$(_MonoAndroidNETDefaultOutDir)ref\Mono.Android.Runtime.dll" />
```

Comment:
I fear that I'm gonna be "bikeshedding" half the naming here -- and I am! -- but…

`Mono.Android.Runtime` is not "future-facing".  We *may* stick with `Mono.Android.dll` "forever", but there are equal odds that we *may* do a `Microsoft.Android.dll` name in the future (.NET 10?).

As such, we should choose names which are unlikely to need changing.  Possible suggestions:

  * `Microsoft.Android.Startup.dll`
  * `Microsoft.Android.Internals.dll`
  * `Microsoft.Android.Interop.dll`

Reply:
I'd rather keep it `Mono.Android.Runtime` for now, since it pairs nicely with `Mono.Android` (and that was the reason I chose this name)